<launch>
	<arg name="veh"/>
	<arg name="param_file_name" default="default" doc="Specify a param file. ex:megaman"/>
	<arg name="pkg_name" default="obstacle_detection" doc="name of the package"/>
	<arg name="node_name" default="obstacle_detection_node" doc="name of the node"/>

        <group ns="$(arg veh)">
	    <node  pkg="$(arg pkg_name)" type="$(arg node_name).py" name="$(arg node_name)" output="screen">
	        <rosparam command="load" file="$(find obstacle_detection)/config/$(arg node_name)/$(arg param_file_name).yaml"/>
            <remap from="obstacle_detection_node/duckie_image" to="camera_node/image/compressed" />
            <remap from="obstacle_detection_node/duckie_image" to="camera_node/image/compressed" />
	    </node>
	</group>
</launch>
